### ISSUE TITLE
recipe breaking changes

### ISSUE THREAD

**Issue by topepo** [erblast/easyalluvial#10]
I'm doing reverse dependencies for `recipes` and saw an error for `easyalluvial`: 

 ```
    > 
    > data = as_tibble(mtcars)
    > categoricals = c('cyl', 'vs', 'am', 'gear', 'carb')
    > numericals = c('mpg', 'cyl', 'disp', 'hp', 'drat', 'wt', 'qsec')
    > max_variables = 5
    > 
    > data = data %>%
    +   mutate_at( vars(categoricals), as.factor )
    > 
    > 
    > alluvial_wide( data = data
    +                 , max_variables = max_variables
    +                 , fill_by = 'first_variable' )
    Error: No role currently exists for column(s): 'easyalluvialid'. Please use `update_role()` instead.
    Execution halted
```

We changed the role system in 0.1.4 that broke this (and updated it a little more in 0.1.5). 

Let me know if you need help. You can test against the current master in GH for `recipes`.

**Comment by erblast** [comment]
thanks for letting me know, this will get fixed

**Comment by topepo** [comment]
I forgot to mention to make the dependency `recipes >= 0.1.5`. I'll submit it tonight and mention this thread.

**Comment by erblast** [comment]
fixed in 47ceb3bd36a5bf5d6b68f08fcb2652173ab3d9a5 and 1a8deb1b82c5abb596806084c1ac3f4b0589e1a6, will submit a new version to CRAN some time next week
